Gemfile.lock

```diff
@@ -258,8 +258,8 @@ GEM
       activerecord (>= 4.2)
       activesupport (>= 4.2)
       after_commit_action (~> 1.0)
-    crack (0.4.3)
-      safe_yaml (~> 1.0.0)
+    crack (0.4.5)
+      rexml
```

Comment:
That's an interesting change. They replaced a YAML library with an XML one.

Reply:
It only appears that way because of the bump from 0.4.3 to 0.4.5 - they removed `safe_yaml` years ago, and just recently added `rexml`:
https://github.com/jnunemaker/crack/commits/master/crack.gemspec